### PR TITLE
bpo-24053: Add EXIT_SUCCESS and EXIT_FAILURE values in the sys module

### PR DESCRIPTION
**Pull request by arnolddumas**
This pull request adds EXIT_SUCCESS and EXIT_FAILURE constants in the sys module.

https://bugs.python.org/issue24053

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Our records indicate we have not received your CLA. For legal reasons we need you to sign this before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.

If you have recently signed the CLA, please wait at least one business day
before our records are updated.


You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for your contribution, we look forward to reviewing it!

File: Doc/library/sys.rst [hunk 1]
```diff
@@ -1594,6 +1594,24 @@ always available.
    .. versionadded:: 3.2
 
 
+.. data:: EXIT_SUCCESS
+
+    An int you can use to indicate your program ended gracefully.
+
+    .. code-block:: python
+
+        sys.exit(sys.EXIT_SUCCESS)
```
Comment (isidentical) [inline Doc/library/sys.rst:1622]: I think `versionadded` should be added for noticing people this is a new value.

**Comment by furkanonder** [comment]
LGTM.

**Review by furkanonder — APPROVED** [review]

**Comment by encukou** [comment]
The issue is closed; I'll close the PR as well.

**Pull request closed without merging**